vmm/src/device_manager/mmio.rs

```diff
@@ -266,6 +276,12 @@ impl MMIODeviceManager {
         None
     }
 
+    #[allow(dead_code)]
```

Comment:
nit: I'd make this into:
```suggestion
    // Currently only used on 'aarch64'.
    #[cfg(target_arch = "aarch64")]
```
since it's not actually dead code.